app/routes/task_routes.py

```diff
@@ -1 +1,151 @@
-from flask import Blueprint
\ No newline at end of file
+from ..models.task import Task
+from flask import Blueprint, abort, make_response, request, Response,jsonify,current_app
+from ..db import db
+from .route_utilities import validate_model,create_model,get_models_with_filters
+from datetime import datetime,timezone
```

Comment:
Spacing issues across imports:
```suggestion
from flask import Blueprint, abort, make_response, request, Response, jsonify, current_app
from ..db import db
from .route_utilities import validate_model, create_model, get_models_with_filters
from datetime import datetime, timezone
```